S3.Rmd

```diff
@@ -54,7 +54,7 @@ getS3method("mean", "Date")
 ### Exercises
 
 1.  The most important S3 objects in base R are factors, data frames,
-    and date/times (Dates, POSIXct, POSIXlt). You've already seen the 
+    and date-times (Dates, POSIXct, POSIXlt). You've already seen the 
     attributes and base type that factors are built on. What base types and 
     attributes are the others built on?
 
```

Comment:
I think the Venn diagram (Q3) might be beneficial for you to include in the book as a figure, rather than an exercise, as it's confusing to keep them straight and a visual might help.